app/views/board/bits.scala

```diff
@@ -14,8 +14,11 @@ object bits {
 
   private val dataState = attr("data-state")
 
+  def color(playerColor: chess.Color, game: lila.game.Game): chess.Color =
```

Comment:
please make it `private`, accept only a single `pov: Pov` argument, and rename it to maybe `miniOrientation`

Reply:
Yes, it looks much better.

Reply:
for some reason, scalafmtAll has modified a portion that had not been modified when I ran scalafmtAll in the first commit